front/src/components/search/SearchContainer.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
-import { Flex, Spinner, Text } from '@chakra-ui/core';
+import {
+  Flex,
+  Spinner,
+  Text,
+  Stat,
+  StatLabel,
+  StatHelpText,
+  Image,
+  StatGroup,
+} from '@chakra-ui/core';
 import { ShowSearchResult } from 'types/external';
 import SearchResults from './subcomponents/SearchResults';
+import logo from '../../images/logo.svg';
 
 interface Props {
   isInputDirty: boolean;
   isLoading: boolean;
   shows: ShowSearchResult[];
   totalResults: number;
+  colorMode: 'light' | 'dark';
```

Comment:
Instead of passing colorMode as a prop, I think we can import colorMode, like SearchPage does, and use it inside the WelcomeMessage component. So this would go inside WelcomeMessage:

`const { colorMode } = useColorMode();`

Reply:
will do